adal/src/main/java/com/microsoft/aad/adal/AcquireTokenWithBrokerRequest.java

```diff
@@ -59,13 +59,13 @@ AuthenticationResult acquireTokenWithBrokerSilent()
         logBrokerVersion();
 
         final AuthenticationResult authenticationResult;
-        if (!StringExtensions.IsNullOrBlank(mAuthRequest.getBrokerAccountName()) || !StringExtensions
-                .IsNullOrBlank(mAuthRequest.getUserId())) {
-            Logger.v(TAG, "User is specified for background(silent) token request, trying to acquire token silently.");
-            authenticationResult = mBrokerProxy.getAuthTokenInBackground(mAuthRequest);
-        } else {
+        if (StringExtensions.isNullOrBlank(mAuthRequest.getBrokerAccountName()) && StringExtensions
```

Comment:
As we talked on Friday, I think we should probably go through all the checkstyle, findbug rules. Those kind of changes for flipping the check, i don't think there are too much needs to do it. Not seeing any benefits we get from here.